dcos_installer/test_backend.py

```diff
@@ -288,7 +265,7 @@ def test_do_configure(tmpdir, monkeypatch):
 """
 
 
-def test_do_aws_configure(tmpdir, monkeypatch):
+def test_do_aws_configure(tmpdir, monkeypatch, release_config_aws):
```

Comment:
argument order. There could be an intuitive preference for `monkeypatch` arg to be the last one.

Reply:
by that same logic, wouldn't `tmpdir` also have to be behind `release_config_aws`? Seems inconsequential unless we have a rule that says "all built-in pytest fixtures should be listed after custom fixtures"